app/Domain/Session.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace Notes\Domain;
+
+use Notes\Exception\ModelNotFoundException as ModelNotFoundException;
+use Notes\Mapper\Session as SessionMapper;
+use Notes\Model\Session as SessionModel;
+use Notes\Domain\User as UserDomain;
+use Notes\Validator\InputValidator as InputValidator;
+
+class Session
+{
+    public function __construct()
+    {
+        $this->validator = new InputValidator();
+    }
+    
+    public function create($userModel, $sessionModel)
+    {
+        
+        $userDomain    = new UserDomain();
+        $userModelRead = $userDomain->readByUserNameAndPassword($userModel);
+        if (!empty($userModelRead)) {
+            $sessionModel->setUserId($userModelRead->getId());
+            
+            $randomNumber = 100;
+
```

Comment:
createdOn should be passed from webserver. Should not be asked database to assign it on itself.